tests/rbac/common/addresser/role_owner_test.py

```diff
@@ -79,18 +79,16 @@ def test_address_random(self):
             addresser.address_is(rel_address2), addresser.AddressSpace.ROLES_OWNERS
         )
 
-    def test_address_static(self):
-        """Tests address makes the expected output given a specific input"""
-        role_id = "99968acb8f1a48b3a4bc21e2cd252e67"
-        user_id = "966ab67317234df489adb4bc1f517b88"
-        expected_address = (
-            "bac00100005555326a1713a905b26359fc8da23333cce7570f3f6f7d2c1635f6deea00"
-        )
-        rel_address = addresser.role.owner.address(
-            object_id=role_id, related_id=user_id
-        )
-        self.assertIsAddress(rel_address)
-        self.assertEqual(rel_address, expected_address)
-        self.assertEqual(
-            addresser.address_is(rel_address), addresser.AddressSpace.ROLES_OWNERS
-        )
+    def test_addresser_parse(self):
```

Comment:
Am I seeing things or is this test repeated across several files? Can we extract it or make most of it generic so the tests don't get out of phase?

Reply:
addressing is over-tested, it can be reduced in a later PR.
Each object type verifies its public addressing methods are as expected